ckanext/datajson/tests/test_datajson_ckan_all_harvester.py

```diff
@@ -140,13 +137,21 @@ def test_datason_usda(self):
         assert_equal(len(dataset.resources), 1)
         assert_in(munge_title_to_name("Congressional Logs"), tags)
 
+    def test_source_returning_http_error(self):
+        url = 'http://www.cpsc.gov/data.json'
```

Comment:
```suggestion
        url = 'http://nonexisting-url.abc/404'
```
Ensure URL not exist now and in the future